covid_graphs/covid_graphs/country_graph.py

```diff
@@ -80,22 +89,54 @@ def __init__(
         self.log_title = f"Days [since {self.log_xaxis_date_since.strftime('%b %d, %Y')}]"
         self.date_title = "Date"
 
-        max_value = max(
+        self.max_value = max(
             max(self.cropped_cumulative_active),
             max(trace.max_value for trace in self.trace_by_event.values()),
         )
         self.log_yrange = [
             math.log10(max(1, self.cropped_cumulative_active.min())) - 0.3,
-            math.log10(max_value) + 0.3,
+            math.log10(self.max_value) + 0.3,
+        ]
+
+    def _create_slider(self, traces: List[Scatter]):
+        SUBTRACE_COUNT = 4
```

Comment:
Minor: All caps are usually for global constants. In this instance, this could be a good solution:
`_create_slider(self, traces: List[Scatter], subtrace_count: int = 4) -> Dict[str, Any]: `

Reply:
Oh I see what this constant is... number of traces per legend item. Maybe it can indeed be moved to the global scope at the top of the file.

I also wonder if the traces can be selected based on `legendgroup` attribute instead.

Reply:
I went with `legendgroup`, which simplifies the logic and is less error-prone.